### PR TITLE
protocols/dcutr: disable `libp2p-core` default features

### PR DESCRIPTION
**Pull request by elenaf9**
# Description

Fix `libp2p-dcutr` is leaking the default features for `libp2p-core`.

## Links to any relevant issues

Workaround for #2833.

### CODE REVIEW

**Comment by elenaf9** [comment]
@mxinden do I also need to bump the dependency version in the root `Cargo.toml` and bump the `libp2p` version or is this not necessary because its only a patch version bump?

File: protocols/dcutr/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.5.1 [unreleased]
```
Comment (mxinden) [inline protocols/dcutr/CHANGELOG.md]:
```suggestion
# 0.5.1
```

I will release this right away.

**Review by mxinden — APPROVED** [review]
Thanks for the fix! Will cut a new release right away.

> @mxinden do I also need to bump the dependency version in the root Cargo.toml and bump the libp2p version or is this not necessary because its only a patch version bump?

No need for the bump in the root `Cargo.toml`.

**Comment by mxinden** [comment]
[Tagged](https://github.com/libp2p/rust-libp2p/releases/tag/libp2p-dcutr-0.5.1) and [published](https://crates.io/crates/libp2p-dcutr/0.5.1).

**Pull request merged**